pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -194,16 +192,13 @@ stdenv.mkDerivation (finalAttrs: {
     bash-completion
     pango
     tpm2-tss
-    efivar
     fwupd-efi
     protobufc
     modemmanager
     libmbim
     libcbor
     libqmi
     xz # for liblzma
-  ] ++ lib.optionals haveDell [
```

Comment:
is haveDell used elsewhere

Reply:
Yeah it is
https://github.com/NixOS/nixpkgs/blob/cfed17c5397808b9f77a7377d8af24c9a332c908/pkgs/os-specific/linux/firmware/fwupd/default.nix#L224-L225

Reply:
Is dell still the right name to gate it under? I'm not sure of the original reasoning but a casual search points to synaptics hardware also being present in other manufacturers' hardware, but maybe the support for firmware updates is limited to Dell here.